powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/configuration/PowerAuthServiceConfiguration.java

```diff
@@ -162,12 +162,19 @@ public class PowerAuthServiceConfiguration {
     private int offlineSignatureComponentLength;
 
     /**
-     * Expiration of timestamps for ECIES and MAC token requests.
+     * Expiration of timestamps for ECIES and MAC token requests for protocol version 3.2+.
      */
     @Value("${powerauth.service.crypto.requestExpirationInMilliseconds}")
     @Min(0)
     private int requestExpirationInMilliseconds;
 
+    /**
+     * Expiration of timestamps for ECIES and MAC token requests for protocol version 3.1 or older.
+     */
+    @Value("${powerauth.service.crypto.requestExpirationInMillisecondsBeforeV32}")
```

Comment:
I would generify the naming a bit, maybe using `*Extended` instead of `*BeforeV32`?

Reply:
Sure.